helm/ph-ee-test/config/application-bb.properties

```diff
@@ -0,0 +1,13 @@
+store.local.interop.host=https://rhino2.mifos.g2pconnect.io
+store.local.customer.host=https://rhino2.mifos.g2pconnect.io
+zeebe.broker.contactpoint=zeebetest-zeebe-gateway:26500
+
+security.oauth2.resource.jwt.key-uri=http://ops-bk2.mifos.g2pconnect.io/oauth/token_key
+rest.authorization.enabled=false
+rest.authorization.host=http://ops-bk2.mifos.g2pconnect.io
+
+dfspids=rhino, gorilla
+
+
```

Comment:
One of these properties file is missing the tenant to use case BPMN configurations that Ankita had added in sprint 3. Please request Ankita to share that configuration.

Reply:
cc: @ankita10r

Reply:
@avikganguly01 Added configs in application-bb.properties file.